docker/ray-deps/README.md

```diff
@@ -14,7 +14,7 @@ Images are `tagged` with the format `{Ray version}[-{Python version}][-{Platform
 | `nightly`                    | The most recent Ray development build (a recent commit from Github `master`) |
 | `6 character Git SHA prefix` | A specific development build (uses a SHA from the Github `master`, e.g. `8960af`). |
 
-The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py37`, `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.7`.
+The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.7`.
```

Comment:
```suggestion
The optional `Python version` tag specifies the Python version in the image. All Python versions supported by Ray are available, e.g. `py38`, `py39` and `py310`. If unspecified, the tag points to an image using `Python 3.7`.
```

the tag points to an image using `Python 3.7`.

this part needs to be changed (is it pointing to python 3.8 now?)